### PR TITLE
🎺  New horn updates

### PR DESCRIPTION
**Pull request by bradp**
See #256

**Updates**
- Correctly counts down & checks for ready state
- Correctly sounds horn from alert
- Check for king's reward (still needs testing)

### CODE REVIEW

File: src/scripts/content.js [hunk 1]
```diff
@@ -56,14 +56,19 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
         let message = "Logged out";
         const krHudElement = document.getElementsByClassName('mousehuntHud-huntersHorn-response')[0];
         const krPageElement = document.getElementsByClassName('mousehuntPage-puzzle-form-state hasPuzzle')[0];
-        const hunt_timer = document.getElementById('huntTimer');
+        const hunt_timer = document.querySelector(".huntersHornView__countdown");
+        const is_ready = document.querySelector('.huntersHornView__timerState--type-ready');
```
Comment (AardWolf) [inline src/scripts/content.js:60]: I think we can just confirm the hunt_timer hasClass('ready')

**Comment by bradp** [comment]
Pushed some changes for it. 

There's room for improvement, but currently working for me as far as I can tell. 👍

File: src/scripts/content.js (hunk 1 above)
Comment (bradp) [reply src/scripts/content.js]: Updated in `7993742bcb21` 👍

**Review by AardWolf — APPROVED** [review]
I've been running with it locally for a while and it appears to work.

**Pull request merged**